libcontainer/configs/mount.go

```diff
@@ -30,6 +30,12 @@ type Mount struct {
 	// Relabel source if set, "z" indicates shared, "Z" indicates unshared.
 	Relabel string `json:"relabel"`
 
+	// RecAttrSet represents mount properties to set recursively (AT_RECURSIVE), see mount_setattr(2)
```

Comment:
Nit: end a sentence with a period.

Reply:
This can _probably_ be generalized nicely by adding something like

```go
Attr *unix.MountAttr
```

(and can be reused later to e.g. change propagation flags etc.)

Reply:
I mean, replace`RecAttrSet` and `RecAttrClr` with `Attr`.

Reply:
`*unix.MountAttr` lacks the field for `AT_RECURSIVE`

Reply:
Updated to use `RecAttr` as the name